backend/api/Controllers/RobotController.cs

```diff
@@ -353,6 +352,16 @@ public async Task<ActionResult<IList<VideoStream>>> GetVideoStreams([FromRoute]
             return NotFound("Robot not found");
         }
 
+        if (robot.Status is RobotStatus.Stuck)
+        {
+            _logger.LogWarning(
+                "Robot '{id}' is stuck ({status})",
+                robotId,
+                robot.Status.ToString()
+            );
+            return Conflict($"The Robot is stuck ({robot.Status})");
+        }
```

Comment:
Suggest removing. Want to keep the option to try a new mission, different mission, if the current one is blocked

Reply:
Agreed.